lib/googleanalytics.jsx

```diff
@@ -0,0 +1,27 @@
+var React = require('react');
+var config = require('./config');
+
+var GoogleAnalyticsBasicTag = React.createClass({
+  render: function() {
+    if (config.GA_ACCOUNT) {
+      return (
+        <script dangerouslySetInnerHTML={{
+          __html: "(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ \
```

Comment:
That's awesome! :+1: 
Absolutely no apologies, or reverts needed here. Thanks for working on that.